README.md

```diff
@@ -23,10 +23,27 @@
 1. Roadmap
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
+It is made to simplify developing data processing pipelines on GCP.
+
+The main features are:
+
+* [Dockerized jobs](./docs/project_setup_and_build.md#overview)
+* [Powerful CLI](./docs/cheatsheet.md)
+* [Automated build](./docs/project_setup_and_build.md#overview), [deployment](./docs/deployment.md), 
+[versioning](./docs/project_setup_and_build.md#project-versioning), [configuration](./docs/configuration.md)
+* [Unified project structure](./docs/project_setup_and_build.md#project-structure)
```

Comment:
Airflow DAG files generator

Reply:
I think we might be the only people in the universe who do that. And it's hard to understand why, without any context. Contrary to, for example, automated build, deployment, and project structure. Those things are universal in tech.